packages/astro/src/runtime/server/astro-global.ts

```diff
@@ -26,28 +19,13 @@ function createAstroGlobFn() {
 // This is used to create the top-level Astro global; the one that you can use
 // Inside of getStaticPaths.
 export function createAstro(
-	filePathname: string,
-	_site: string | undefined,
-	projectRootStr: string
+	_filePathname: string, // legacy unused param from compiler
+	site: string | undefined,
+	_projectRootStr: string // legacy unused param from compiler
```

Comment:
I assume this is a separate TODO for the compiler?

Reply:
Good idea. Let's put a TODO here and make sure we make all the breaking changes for the compiler as we're planning a 1.0 for it too.